egs/multi_conv_zh_es_ta/ST/prepare.sh

```
@@ -0,0 +1,175 @@
+  ./local/compute_fbank_gpu.py
+  ./local/compute_fbank_gpu.py --test
+fi
+
+if [ $stage -le 4 ] && [ $stop_stage -ge 4 ]; then
+  log "Stage 4: Compute fbank for musan"
+  ./local/compute_fbank_musan.py
+fi
+
+if [ $stage -le 6 ] && [ $stop_stage -ge 6 ]; then
+  log "Stage 6: Prepare BPE based lang"
+  for vocab_size in ${vocab_sizes[@]}; do
```

Shellcheck (0.10.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)